pages/common/steam.md

```diff
@@ -11,6 +11,10 @@
 
 `steam -console`
 
+- Enable the Steam console tab while Steam is running:
```

Comment:
Does this open or just enable the console tab?

Reply:
Enables and opens.

Reply:
```suggestion
- Enable and open the Steam console tab while Steam is running:
```
This would also be an option, but only if you like:
```suggestion
- Enable and open the Steam console tab in a running Steam instance:
```